engine/consensus/dkg/reactor_engine_test.go

```diff
@@ -134,3 +136,68 @@ func TestEpochSetup(t *testing.T) {
 	controller.AssertExpectations(t)
 	keyStorage.AssertExpectations(t)
 }
+
+// TestReactorEngine_EpochCommittedPhaseStarted ensures that we are logging
+// a warning message whenever we have a mismatch between the locally produced DKG keys
+// and the keys produced by the DKG smart contract.
+func TestReactorEngine_EpochCommittedPhaseStarted(t *testing.T) {
+	rand.Seed(time.Now().UnixNano())
+	currentCounter := rand.Uint64()
+	nextCounter := currentCounter + 1
+	me := new(module.Local)
+
+	privKey, _ := unittest.NetworkingKey()
```

Comment:
```suggestion
	privKey, _ := unittest.StakingKey()
```
So that we are testing with a BLS key, which is what is generated by the DKG, rather than ECDSA.